.github/workflows/build-test.yml

```diff
@@ -13,8 +13,8 @@ on:
 jobs:
   build:
     env:
-      NX_BRANCH: ${{ github.ref }}
-      NX_RUN_GROUP: ${{ github.run_id }}-${{ github.run_number }}
+      NX_BRANCH: ${{ github.event.number }}
+      NX_RUN_GROUP: ${{ github.action }}
```

Comment:
Github returns nothing though, `$GITHUB_RUN_ID` would be better.
You can find the info here: https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables